packages/extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logion/extension",
-  "version": "0.4.4-2",
+  "version": "0.4.4-3",
```

Comment:
This is not required, you can actually upgrade logion/client in the wallet without having to bump extension's version.

Reply:
OK. Change removed.